_includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/produce-topic-keys-values.adoc

```diff
@@ -0,0 +1,19 @@
+To get started, lets produce some records to your new topic.
+
+Since you've created a topic with more than one partition, you'll send full key-value pairs so you'll be able to see how different keys end up on different partitions.
+To send full key-value pairs you'll specify the `parse.key` and `key.separator` options to the console producer command.
+
+Let's run the following command in the broker container shell from the previous step to start a new console producer:
```

Comment:
At this point we don't have any containers running so "broker container shell" should probably be changed to just CCloud CLI producer. Same with on line 19, talking about the broker container.

Reply:
Good catch, fixed in latest commit